### PR TITLE
Add static render function

### PR DESCRIPTION
**Pull request by sjdemartini**
In order to improve performance when using static-rendering (i.e., server-side only static markup with no need for client-side functionality), a static-rendering version of `renderReactWithAphrodite` is added here. It is modeled after `hypernova-react`'s static renderer: https://github.com/airbnb/hypernova-react/blob/5de9af0d49a758f5a536f1f3b273b37434e230fb/src/index.js#L29-L35

### CODE REVIEW

File: src/index.js
Comment (sjdemartini) [inline src/index.js]: @goatslacker Does using `html` here directly seem okay, as opposed to using the `markup` returned from `serialize`? I would assume so, but want to make sure there isn't a security hole in not using `serialize`, for instance.

Comment (goatslacker) [reply src/index.js]: Should be fine. `serialize` is for sending down the JSON to the client which we don't do when static rendering.

Comment (goatslacker) [reply src/index.js]: Although I think we may need to send down the classes from aphrodite

Comment (sjdemartini) [reply src/index.js]: Thanks for commenting Josh. Why do we need the classes from aphrodite in the output string? Based on the non-static rendered, it seems like those are only used so that the client can be initialized properly https://github.com/airbnb/hypernova-aphrodite/blob/fece1bbf9d689d86a67f915d469c7e7b05b247ec/src/index.js#L24-L25
which isn't necessary for the static-renderer. cc @lencioni

**Comment by sjdemartini** [comment]
Thoughts @goatslacker, @ljharb, @lencioni? Would be great to get this in and start using it, if this looks good to you

**Review by lencioni — APPROVED** [review]

File: src/index.js
Comment (ljharb) [inline src/index.js]: we should probably use `data-aphrodite="data-aphrodite"` here, for max compat with pre-HTML5. probably not a big deal tho.

**Review by ljharb — APPROVED** [review]
I think we should wait for @goatslacker's stamp tomorrow, but this LGTM.

File: src/index.js
Comment (sjdemartini) [reply src/index.js]: This mimics what's done in the non-static renderer. I can change it to `data-aphrodite="data-aphrodite"` for both, though, if you'd like that to happen in this PR.

Comment (ljharb) [reply src/index.js]: I don't think it's a blocker, but I think it would be a good change in both places :-)

**Comment by sjdemartini** [comment]
Alright, updated to use `data-aphrodite="data-aphrodite"`. Should we wait for @goatslacker's stamp, or is this good to go as is?

**Comment by lencioni** [comment]
:shipit:

**Comment by ljharb** [comment]
I was hoping this could wait for @goatslacker's stamp, per my approval message.

**Comment by lencioni** [comment]
@ljharb ah gotcha. Let's wait for him to chime in before publishing then.

**Comment by goatslacker** [comment]
🚢

**Pull request merged**
